react/index.js

```diff
@@ -1,27 +1,63 @@
-import React from 'react'
+import React, {Component} from 'react'
 import PropTypes from 'prop-types'
-import {graphql} from 'react-apollo'
-import query from './query.graphql'
-
-const Library = ({data: {loading, books, sampleStaticResolver}}) => {
-  return (
-    <div>
-      <h3>My Library</h3>
-      {loading && 'loading'}
-      {!loading && books.map(({id, name, author}) => <div key={id}>{name} - {author}</div>)}
-      {!loading && sampleStaticResolver.result}
-    </div>
-  )
+import {compose, graphql} from 'react-apollo'
+
+import createAutopersistedBook from './graphql/createAutopersistedBook.graphql'
+import deleteAutopersistedBook from './graphql/deleteAutopersistedBook.graphql'
+import createCachedBook from './graphql/createCachedBook.graphql'
+import deleteCachedBook from './graphql/deleteCachedBook.graphql'
+import getBooks from './graphql/getBooks.graphql'
+
+import {BooksManager} from './booksManager'
+
+class Library extends Component {
+  constructor(props) {
```

Comment:
There's no need for this constructor, it will work the same way if you just remove it :)